src/Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI/Service/Impl/CustomOpenApiWriter.cs

```diff
@@ -0,0 +1,21 @@
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI.Extension;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI.Service.Interface;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Enums;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Extensions;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI.Service.Impl
```

Comment:
Can we change the folder structure from `Service.Impl` to `Services` and also change the namespace accordingly?

Reply:
Changed to match request in new commit